twitter-server/src/com/treode/twitter/finagle/http/package.scala

```diff
@@ -125,14 +173,20 @@ package object http {
         TxClock.parse (value) .getOrThrow (new BadRequestException (s"Bad time for $name: $value"))
       }
 
+	//Maintaining compatibility with un-updated servers that use older headers
```

Comment:
I think finatra and unfiltered are self contained and won't be affected by this...I think. Make the change and the run their tests to find out.

Yeah, change the method names now. For the Condition-TxClock that will need different defaults, make the default a parameter.